Validation/RecoTrack/python/plotting/plotting.py

```diff
@@ -2532,6 +2532,7 @@ def create(self, tdirectoryNEvents, requireAllHistograms=False):
     def draw(self, *args, **kwargs):
         kargs = copy.copy(kwargs)
         kargs["ratio"] = False
+        kargs["separate"] = False
```

Comment:
Could you explain why this change is needed?

Reply:
Ciao Matti,
the separate option was giving conflicts (crashes, IIRC) if coupled with the PlotOnSideGroup, which sort of makes sense since the 2 concepts are somehow "opposite". That's why I defaulted that option to False in this specific case. I guess you never tested the 2 coupled together, right?

Reply:
It's indeed very likely that I did not test these two together, and on a further thought I do agree that the "separate" concept doesn't really apply to the PlotOnSideGroup. Thanks for the explanation!